lib/membrane_rtsp/parser.ex

```diff
@@ -0,0 +1,62 @@
+defmodule Membrane.RTSP.Parser do
+  @moduledoc false
+
+  alias __MODULE__.{Request, Transport}
+  alias Membrane.RTSP
+
+  @type transport_header :: [
+          transport: :TCP | :UDP,
+          mode: :unicast | :multicast,
+          parameters: map()
+        ]
```

Comment:
I don't know what's going on, but whenever I run `mix docs` I get
```
warning: documentation references type "Membrane.RTSP.Parser.transport_header()" but it is hidden or private
  (membrane_rtsp 0.5.3) lib/membrane_rtsp/request.ex:111: Membrane.RTSP.Request.parse_transport_header/1

```
The type is defined correctly, though... Can you reproduce this?

Reply:
Yes the reason for this is that we reference a type definition in module with `@moduledoc false`